### ISSUE TITLE
VDirectiveKey: parsing arguments changes argument into lowercase letters.

### ISSUE THREAD

**Issue by armano2** [mysticatea/vue-eslint-parser#12]
In all `VDirectiveKey` `argument` is lowercased.
required for: vuejs/eslint-plugin-vue#95

Tested on version: 2.0.0-beta.2

**Input**
```html
<template>
    <div v-bind:FooBar=""></div>
</template>
```
**Output**
```js
...
{
    "type": "VDirectiveKey",
    "range": [
        21,
        34
    ],
    "loc": {
        "start": {
            "line": 2,
            "column": 9
        },
        "end": {
            "line": 2,
            "column": 22
        }
    },
    "name": "bind",
    "argument": "foobar",
    "modifiers": [],
    "shorthand": false
}
...
```

**Comment by mysticatea** [comment]
Thank you for this report.

In except the defined properties of custom components, `v-on:attrName` and `v-on:attrname` are the same attribute, both become `v-on:attrname`. So this is not a bug. However, I agree it's inconvenient if we cannot get the raw data of `key.argument` easily. I will add new property `raw`.

```js

{
    "type": "VDirectiveKey",
    "range": [
        21,
        34
    ],
    "loc": {
        "start": {
            "line": 2,
            "column": 9
        },
        "end": {
            "line": 2,
            "column": 22
        }
    },
    "name": "bind",
    "argument": "foobar",
    "modifiers": [],
    "raw": {
        "name": "bind",
        "argument": "FooBar",
        "modifiers": [],
    },
    "shorthand": false
}
```

**Comment by armano2** [comment]
@mysticatea thank you, its going to be easier to get it from raw than accessing source code & parsing it by my own.

**Comment by mysticatea** [comment]
I added some properties (https://github.com/mysticatea/vue-eslint-parser/compare/v2.0.0-beta.2...master#diff-60a3c5d4caa5a12bdd8e920349affeb7). Please use `v2.0.0-beta.3`.

- `VIdentifier#rawName`
- `VDirectiveKey#raw`
- `VElement#rawName`

**Comment by armano2** [comment]
@mysticatea thank you
